src/index.html

```diff
@@ -51,5 +50,43 @@ <h1 class="hero-header">Discover Amazing places in Japan</h1>
 
     </div>
 
+
+    <!-- Begin benefits section -->
+    <div class="benefits-section">
+        <h3 class="benefits-header">Benefits of Odigo</h3>
+
+        <div class="benefits-content">
+            <div class="card-item">
+                <img src="/assets/icons/Icon-benefits1.png" alt="temple">
+                <h6 class="card-header">Welcome to Odigo!</h6>
+                <p class="card-description">Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over.</p>
+                <a href="#" class="card-detail" >
```

Comment:
please use javascript:void(0) for href with a tag not have any action

Reply:
thanks for your feedback and suggestions